metadata-ingestion/tests/integration/grafana/test_grafana.py

```diff
@@ -273,6 +282,51 @@ def verify_grafana_fully_ready(
     logging.warning(f"Grafana may not be fully ready after {timeout}s timeout")
 
 
+def verify_grafana_entities_provisioned(timeout: int = 180) -> None:
```

Comment:
Maybe use tenacity here instead of manually adding retry logic? We already use it in our code base. Otherwise looks good